docs/apache-airflow-providers-amazon/operators/sagemaker.rst

```diff
@@ -208,6 +208,20 @@ The output is placed in an S3 bucket, and automatically deployed if configured f
     :start-after: [START howto_operator_sagemaker_auto_ml]
     :end-before: [END howto_operator_sagemaker_auto_ml]
 
+.. _howto/operator:SageMakerCreateExperimentOperator:
```

Comment:
This seems to be an unrelated change -- could you pull it out in to a separate PR please?